docs/en_US/TrialExample/Pix2pixExample.rst

```diff
@@ -0,0 +1,120 @@
+
+**Goals**
+^^^^^^^^^^^^^
+
+Although GANs are known to be able to generate high-resolution realistic images, they are generally fragile and difficult to optimize, and mode collapse can happen during training due to improper optimization setting, loss formulation, model architecture, weight initialization, or even data augmentation patterns. The goal of this tutorial is to leverage NNI to automatically find a good setting for these important factors. 
+
+In this example, we have selected the following hyperparameters:
+
+
+* ``ngf``: number of generator filters in the last conv layer  
+* ``ndf``: number of discriminator filters in the first conv layer
+* ``netG``: generator architecture
+* ``netD``: discriminator architecture
+* ``norm``: normalization type
+* ``init_type``: weight initialization method
+* ``lr``: initial learning rate for adam
+* ``beta1``: momentum term of adam
+* ``lr_policy``: learning rate policy
+* ``gan_mode``: type of GAN objective
+* ``lambda_L1``: weight of L1 loss in the generator objective
+
+
+**Experimental**
+^^^^^^^^^^^^^^^^^^^^
+
+Preparations
+^^^^^^^^^^^^
+
+This example requires the GPU version of PyTorch. PyTorch install package should be chosen based on system, python version, and cuda version.
+
+Please refer to the detailed instruction of installing `PyTorch <https://pytorch.org/get-started/locally/>`__ 
+
+
+Pix2pix with NNI
+^^^^^^^^^^^^^^^^^
+
+**Search Space**
+
+We summarize the range of values for each hyperparameter mentioned above into a single search space json object. 
+
+.. code-block:: json
+		
+   {
+       "ngf": {"_type":"choice","_value":[16, 32, 64, 128, 256]},
+       "ndf": {"_type":"choice","_value":[16, 32, 64, 128, 256]},
+       "netG": {"_type":"choice","_value":["resnet_9blocks", "unet_256"]},
+       "netD": {"_type":"choice","_value":["basic", "pixel", "n_layers"]},
+       "norm": {"_type":"choice","_value":["batch", "instance", "none"]},
+       "init_type": {"_type":"choice","_value":["xavier", "normal", "kaiming", "orthogonal"]},
+       "lr":{"_type":"choice","_value":[0.0001, 0.0005, 0.001, 0.005, 0.01, 0.1]},
+       "beta1":{"_type":"uniform","_value":[0, 1]},
+       "lr_policy": {"_type":"choice","_value":["linear", "step", "plateau", "cosine"]},
+       "gan_mode": {"_type":"choice","_value":["vanilla", "lsgan", "wgangp"]} ,
+       "lambda_L1": {"_type":"choice","_value":[1, 5, 10, 100, 250, 500]}
+    }
+
+Implemented code directory: :githublink:`search_space.json <examples/trials/pix2pix-pytorch/search_space.json>`
+
+Note: starting from v2.0, the search space is directly included in the config, please find the example here: :githublink:`config_v2.yml <examples/trials/pix2pix-pytorch/config_v2.yml>`
```

Comment:
We are already in v2.0. Why don't we keep `config_v2.yml` only?

Reply:
We can do that. I will remove the v1 config.